### PR TITLE
Setup pm2 to launch two instances

### PR DESCRIPTION
**Pull request by j-matus**
Setup pm2 to launch a main instance and a backup instance running on
different ports. If process.env.PORT is provided, the server will
listen on that port instead of the one specified in the server config.
Since the server config object is not writable, its port could not be
directly overwritten. Add an instance config file to avoid warnings.
The build script now deletes the old build and the start script doesn't
call build anymore.

### CODE REVIEW

File: src/server.js [hunk 1]
```diff
@@ -68,7 +68,7 @@ async function createServer() {
 
   configCleanup(db, logger)
 
-  server.listen(serverConfig.port, () => {
+  server.listen(process.env.PORT || serverConfig.port, () => {
```
Comment (refi93) [inline src/server.js]: I would expect the right port to be loaded in serverConfig.port already. Why is this needed?

Comment (j-matus) [reply src/server.js]: An error occurred when trying to reassign `serverConfig.port` since the`serverConfig` object was not writable. If we don't use a separate config file for the backup instance, the port number will be the same for both instances and we need to somehow make sure they will listen on different ports.

**Review by refi93 — APPROVED** [review]
